docs/source/add_new_adaptor.md

```diff
@@ -0,0 +1,256 @@
+
+
+How to Add An Adaptor
+=======
+
+
+- [Introduction](#introduction)
+- [API List that Need to Implement](#api-list-that-need-to-implement)
+- [Add query_fw_capability to Adaptor](#add-query-fw-capability-to-adaptor)
+- [Add Pre-quantization Optimizations](#add-pre-quantization-optimizations)
+- [Quantize Model according to tune cfg](#quantize-model-according-to-tune-cfg)
+- [Evaluate Model on Validation Dataset](#evaluate-model-on-validation-dataset)
+- [Other API that not Mandatory Needed](#other-api-that-not-mandatory-needed)
+- [Summary](#summary)
+
+## Introduction
+Intel® Neural Compressor builds the low-precision inference solution on popular deep learning frameworks such as TensorFlow, PyTorch, MXNet, Keras and ONNX Runtime. The adaptor layer is the bridge between the tuning strategy and vanilla framework quantization APIs.
+
+The document outlines the process of adding support for a new adaptor, in Intel® Neural Compressor with minimal changes. It provides instructions and code examples for implementation of a new adaptor. By following the steps outlined in the document, users can extend Intel® Neural Compressor's functionality to accommodate new adaptor and incorporate it into quantization workflows.
+
+The quantizable operator behavior and it's tuning scope is defined in specific framework YAML file. The adaptor will parse this file and give the quantization capability to the Strategy object. Then Strategy will build tuning space of the specific graph/model and generate different tuning configuration from the tuning space to adaptor.
+
+The diagram below illustrates all the relevant steps of how adaptor is invoked, with additional details provided for each annotated step.
+```mermaid
+  sequenceDiagram
+  	autonumber
+  	autonumber
+    Adaptor ->> Adaptor: Design the framework YAML and inherit `QueryBackendCapability` class
+    Adaptor ->> Adaptor: Initialize the inherited QueryBackendCapability class as self.query_handler
+    Strategy ->> Adaptor: use query_framework_capability to get the capability of the model
+    Adaptor ->> Adaptor: Use self.query_handler to parse the framework YAML and get_quantization_capability
+    Adaptor ->> Adaptor: Pre-optimize the input fp32 model as self.optimized_model
+    Adaptor ->> Strategy: Send the capability including 'opwise' and 'optypewise' ability
+    Strategy ->> Strategy: Build tuning space
+    loop Traverse tuning space
+    	Strategy->> Adaptor: generate next tuning cfg
+        Adaptor ->> Adaptor: calibrate and quantize model based on tuning config
+
+    end
+```
+1. **Adaptor**: Design the framework YAML, inherit `QueryBackendCapability` class to parse the framework yaml. 
```

Comment:
still like what I mentioned, this index + *Adaptor* is not a good way to describe what happens in the diagram. 

is it better to use ① like symbol and skip the first word like **adaptor**?